Files/ViewModels/ItemViewModel.cs

```diff
@@ -36,7 +37,7 @@
 
 namespace Files.ViewModels
 {
-    public class ItemViewModel : INotifyPropertyChanged, IDisposable
+    public class ItemViewModel : ObservableObject, IDisposable
```

Comment:
Is this change related to the current pull request?

Reply:
> Additionally, replaced INotifyPropertyChanged interface in ItemViewModel with toolkit's ObservableObject

Reply:
I'll let this one pass, but in the future, pull requests will not be approved if they have unrelated changes.